validator/keymanager/v2/derived/mnemonic.go

```diff
@@ -0,0 +1,65 @@
+package derived
```

Comment:
Only a derived keymanager needs mnemonic functionality. Will remove the functionality from the direct keymanager in another PR.